### ISSUE TITLE
The current dimensions already are super empty and bland.

### ISSUE THREAD

**Issue by theStrobilus** [Tslat/Advent-Of-Ascension#890]
If I am telling correctly, the next updates will include even more dimensions, but I personally think that's a terrible idea. The current twenty-ish dimensions in the mod are super bland and empty. Take a look at something like the Aether I mod, a relatively simple dimension mod by today's standards, but it still has content that makes you want to stay in it for more than twenty minutes.

Advent's dimensions are way lower on the scale than the original Aether, and that's sort of sad. Every time I enter a dimension, I see a few interesting mobs, maybe a really small dungeon, and that's it. No dimension-specific bosses or anything. I propose the idea of maybe adding per-dimension bosses to each and every dimension, and also using them to create some form of progression chart.

Basically, to reach a different dimension, you would first need to beat the boss (or multiple) in another dimension to, "complete" that dimension. This is a really long-term feature that I doubt can be implemented easily, but I do think it's worthwhile to do so.

Adding more dimensions will just create more and more bland canvases, so I think the current dimensions should be perfected and fun to stay in individually. I mean, I would LOVE to see some giant clown boss in Celeve or something.
